scope/static/js/custom.js

```diff
@@ -87,4 +87,35 @@ $(document).ready(function() {
       $(this).closest('form').submit();
     }
   });
+
+  /* Remove digital file filter(s) on filter tag click. */
+  $('body').on('click', '.digital-file-filter-tag', function () {
+    var $this = $(this);
+    var url = new URL(document.location);
+    var param = $this.data('removeParam');
+    // Special case for dates
+    if (param == 'dates') {
```

Comment:
I think this should probably be a strict comparison `===`, just on the basis of "avoid using ==".

Reply:
Sound good, thanks @cole!